demo/CollapsibleGroup.demo.js

```diff
@@ -1,67 +1,98 @@
 import React, { Fragment } from 'react';
-import { CollapsibleGroup } from '../src'; // swap '../src' for '../dist/build.bundle' to test production
+import { CollapsibleGroup, Collapsible } from '../src'; // swap '../src' for '../dist/build.bundle' to test production
 
 const CollapsibleGroupDemo = () => (
   <Fragment>
     <code>
       <pre>
-        {'<CollapsibleGroup />'}
-        <br />
-        Props:
-        <br />
-        allowMultiple: true
-        <br />
-        collapsibles: array
+        {'<CollapsibleGroup'}
+        <ul style={{ listStyle: 'none' }}>
+          <li>
+            {'allowMultiple={true}'}
+          </li>
+        </ul>
+        {'>'}
       </pre>
     </code>
     <CollapsibleGroup
       allowMultiple
       classPrefix="custom"
-      collapsibles={[
-        {
-          openOnInit: true,
-          clickableNode: <span>click me (click disabled)</span>,
-          children: <span>4 some content</span>,
-          disableClick: true,
-        },
-        {
-          openOnInit: true,
-          clickableNode: <span>click me</span>,
-          children: <span>4 some content</span>,
-        },
-      ]}
-    />
+    >
+      <Collapsible
+        clickableNode={<span>click me</span>}
```

Comment:
Bump these props to one line, since they're singular. The ESLint config should probably enforce this.